plugins/inventory/nmap.py

```diff
@@ -21,6 +21,10 @@
             description: token that ensures this is a source file for the 'nmap' plugin.
             required: True
             choices: ['nmap', 'community.general.nmap']
+        sudo:
+            description: Set to C(true) to execute a C(sudo nmap) plugin scan.
+            version_added: 4.8.0
+            type: boolean
```

Comment:
One thing that makes it a bit nicer in the documenation:
```suggestion
            type: boolean
            default: false
```

Reply:
(Changed order since GitHub UI seems to have a bug and won't merge it in the original order...)